fuse_examples/segmentation/create_dataset.py

```diff
@@ -0,0 +1,169 @@
+# from fastai.vision import *
+import pydicom
+from pathlib import Path
+import pandas as pd
+from tqdm import tqdm as progress_bar
+import PIL
+import numpy as np
+import matplotlib.pylab as plt
+# from mask_functions import *
+# Here out RLE encoding is little bit different
+
+import sys
+sys.path.append('JustImage/')
```

Comment:
You need to install dependencies (add them to requirements) and not to add the code to PYTHONPATH.